src/player.js

```diff
@@ -844,12 +844,24 @@ export default class Player extends FakeEventTarget {
    * @returns {Array<number>} - The possible playback speeds speed of the video.
    */
   get playbackRates(): Array<number> {
-    if (this._engine) {
+    if (this._playbackRates) {
+      return this._playbackRates;
+    } else if (this._engine) {
       return this._engine.playbackRates;
     }
     return [];
   }
 
+  /**
+   * Sets the playbackRates property.
+   * @param {number} playbackRates - Set the possible playback speeds speed of the video.
+   */
+  set playbackRates(playbackRates: number): void {
```

Comment:
where's this using?

Reply:
Forgot that `_handlePlaybackOptions` removed.